themes/bootstrap3/templates/_ui/components/menu-button.phtml

```diff
@@ -0,0 +1,30 @@
+<?php
+  $wrapperTag = $this->wrapperTag ?? 'li';
+  $wrapperClass = $this->wrapperClass ?? 'li';
+
+  $toggleTag = $this->toggleTag ?? 'button';
+  $toggleAttrs = $this->toggleAttrs ?? [];
+
+  $href = $this->href ?? $toggleTag == 'button' ? null : '#';
+?>
+
+<<?=$wrapperTag ?> class="<?=$wrapperClass ?> dropdown">
+  <<?=$toggleTag ?>
+    class="dropdown-toggle"
+    data-toggle="dropdown"
+    <?php if ($href): ?>href="<?=$href ?>"<?php endif ?>
```

Comment:
Do we need to escape `$href` too? In what situation would this be passed in from the outside? (This doesn't seem to be used in existing code here).

Reply:
The theme switcher and language dropdowns work off of hrefs with GET parameters.

Reply:
Isn't it conceivable that the href could contain an `&` character that would need to be HTML escaped? (I realize that is probably not the case in the current applications of the template, but it seems like something that might happen in the future). Adding an escape to be safe seems wise, unless the data is coming in pre-escaped.